src/core/routes/signin.ts

```diff
@@ -7,7 +7,7 @@ import { Account, User } from "../.."
 /** Handle requests to /api/auth/signin */
 export default async function signin(params: {
   options: InternalOptions<"oauth" | "email">
-  query: IncomingRequest["query"]
+  query: IncomingRequest["query"] & { nextauth: string[] }
```

Comment:
I don't think this is necessary

Reply:
Typescript complains for me but might be local development issue only. Would not be the first time.

Reply:
See my recommendation. If you do the necessary change much higher up, this file won't even have to change.

Reply:
Fixed